app/presenters/national_applicability.rb

```diff
@@ -23,8 +23,8 @@ def applies_to
 
 private
 
-  def translated_format_name(count)
-    I18n.t("content_item.format.#{format}", count: count).downcase
+  def translated_schema_name(count)
+    I18n.t("content_item.format.#{schema_name}", count: count).downcase
```

Comment:
Yes, probably. It does feel a little more risky though, and afaik they were imported from Whitehall originally so I'm a bit hesitant to touch them as part of this PR.